instrumentation/opentelemetry-instrumentation-mysql/tests/test_mysql_integration.py

```diff
@@ -117,3 +117,25 @@ def test_uninstrument_connection(self, mock_connect):
 
         spans_list = self.memory_exporter.get_finished_spans()
         self.assertEqual(len(spans_list), 1)
+
+    @mock.patch("mysql.connector.connect")
+    # pylint: disable=unused-argument
+    def test_sqlcommenter_enabled(self, event_mocked):
+        cnx = mysql.connector.connect(database="test")
+        MySQLInstrumentor().instrument(enable_commenter=True)
```

Comment:
Move the MySQLInstrumentor().instrument before the connect,
and consider expanding connect_and_execute_query so you can use it here.

Reply:
Change implemented !!